multicluster/service-mirror/cluster_watcher.go

```diff
@@ -192,18 +192,27 @@ func (rcsw *RemoteClusterServiceWatcher) originalResourceName(mirroredName strin
 	return strings.TrimSuffix(mirroredName, fmt.Sprintf("-%s", rcsw.link.TargetClusterName))
 }
 
-func (rcsw *RemoteClusterServiceWatcher) getMirroredServiceLabels() map[string]string {
-	return map[string]string{
-		consts.MirroredResourceLabel:  "true",
-		consts.RemoteClusterNameLabel: rcsw.link.TargetClusterName,
+func (rcsw *RemoteClusterServiceWatcher) getMirroredServiceLabels(remoteService ...*corev1.Service) map[string]string {
```

Comment:
It's more in common with the rest of the codebase for this to take a pointer rather than a variable number of arguments. Then, we can just check `remoteService != nil`.

They achieve the same thing, but there is never a reason multiple services would be passed into this function so it's better to statically restrict that from happening.

```suggestion
func (rcsw *RemoteClusterServiceWatcher) getMirroredServiceLabels(remoteService *corev1.Service) map[string]string {
```